model/websocket_client.go

```diff
@@ -193,8 +195,6 @@ func (wsc *WebSocketClient) Listen() {
 
 			rawMsg = buf.Bytes()
```

Comment:
I would maybe move this right before we use it (and possibly remove the var declaration on top) or directly pass `buf.Bytes()` to `json.Unmarshal()`.